frontend/packages/noobaa-storage-plugin/src/components/status-card/statuses.tsx

```diff
@@ -1,77 +1,45 @@
-import * as _ from 'lodash';
+import { PrometheusHealthHandler } from '@console/plugin-sdk';
 import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
-import { PrometheusResponse } from '@console/internal/components/graphs';
-import { FirehoseResult } from '@console/internal/components/utils';
 import { getGaugeValue } from '../../utils';
 
-const NooBaaStatus = [
-  {
-    state: HealthState.ERROR,
-    message: 'MCG is not running',
-  },
-  {
+const nooBaaStatus = {
```

Comment:
Using an array is fine. If the keys are not meaningful data.

Reply:
This design looks wrong to me. I would prefer a switch to hold all the information related to statuses or at least an array than an Object with keys whose value does not make sense. The design of not having all the correlated items together does not look right to me.

Reply:
https://github.com/openshift/console/pull/3148#discussion_r438825605